lib/mayaUsd/ufe/StagesSubject.cpp

```diff
@@ -236,6 +236,13 @@ void StagesSubject::stageChanged(UsdNotice::ObjectsChanged const& notice, UsdSta
 				Ufe::Scene::notifyObjectDelete(notification);
 			}
 		}
+#if UFE_PREVIEW_VERSION_NUM >= 2015
+		else if (!prim.IsValid())
+		{
+			auto notification = Ufe::ObjectDestroyed(ufePath);
+			Ufe::Scene::notifyObjectDelete(notification);
+		}
+#endif
```

Comment:
New scene notification added in Ufe. When this message is sent the object is no longer valid. Thus we don't use the scene item, but instead just the path.